tick/base/array/src/array2d.h

```diff
@@ -1,8 +1,6 @@
 #ifndef TICK_BASE_ARRAY_SRC_ARRAY2D_H_
 #define TICK_BASE_ARRAY_SRC_ARRAY2D_H_
 
-// License: BSD 3 clause
```

Comment:
We should replace view_row with the operator() in all places where we only use the view to access single values. Like, `view_row(...)[i]` is bad, because we construct and destroy a view for a single memory access. Also prevents compiler optimizations.